ui/hooks/useCollectiblesCollections.js

```diff
@@ -47,8 +47,7 @@ export function useCollectiblesCollections() {
           );
           newCollections[collectible.address] = {
             collectionName: collectionContract?.name || collectible.name,
-            collectionImage:
-              collectionContract?.logo || collectible.collectionImage,
+            collectionImage: collectionContract?.logo || collectible.image,
```

Comment:
Do we know what `collectible.collectionImage` was?

Reply:
Unfortunately no, `collectionImage` doesn't exist in collectibles object.